packages/at_commons/lib/src/verb/syntax.dart

```diff
@@ -125,7 +125,20 @@ class VerbSyntax {
   static const info = r'^info(:brief)?$';
   static const noOp = r'^noop:(?<delayMillis>\d+)$';
   static const notifyRemove = r'notify:remove:(?<id>[\w\d\-\_]+)';
-  static const enroll =
-      r'^enroll:(?<operation>request|approve|deny):(?:(?:enrollmentId:(?<enrollmentId>[a-zA-Z0-9-]+))|(?:appName:(?<appName>[a-zA-Z0-9]+):)?(?:deviceName:(?<deviceName>[a-zA-Z0-9_]+):)?(?:namespaces:\[(?<namespaces>[a-zA-Z0-9;_,]+)\]:)?(?:totp:(?<totp>[0-9]+):)?(?:apkamPublicKey:(?<apkamPublicKey>.*)))?$';
+  static const enroll = r'^enroll:(?<operation>request|approve|deny):'
+      r'(?:(?:enrollmentId:(?<enrollmentId>[a-zA-Z0-9-]+))|'
+      r'(?:appName:(?<appName>[a-zA-Z0-9]+):)?'
+      r'(?:deviceName:(?<deviceName>[a-zA-Z0-9_]+):)?'
+      r'(?:namespaces:\[(?<namespaces>[a-zA-Z0-9;_,]+)\]:)?'
+      r'(?:totp:(?<totp>[0-9]+):)?(?:apkamPublicKey:(?<apkamPublicKey>.*)))?$';
```

Comment:
What is the purpose of the `(?:` at the start of the various regex fragments in the lines above?

Reply:
?: is for non-capturing group
e.g
deviceName:pixel will capture deviceName:pixel in the group value
?:deviceName:pixel will capture only pixel in the group value.